spec/unit/spark_api/request_spec.rb

```diff
@@ -240,13 +240,22 @@ def version()
         MultiJson.default_adapter.should eq(:yajl) unless jruby?
         result = subject.get('/listings/1000')[0]
         result["StandardFields"]["BuildingAreaTotal"].should be_a(Float)
-        pending("our JSON parser does not support large decimal types.  Anyone feel like writing some c code?") do
-          result["StandardFields"]["BuildingAreaTotal"].should be_a(BigDecimal)
-          number = BigDecimal.new(result["StandardFields"]["BuildingAreaTotal"].to_s)
-          number.to_s.should eq(BigDecimal.new("0.000000000000000000000000001").to_s)
-          number = BigDecimal.new(result["StandardFields"]["ListPrice"].to_s)
-          number.to_s.should eq(BigDecimal.new("9999999999999999999999999.99").to_s)
-        end
+        #   The semantics of `RSpec::Core::Pending#pending` have changed in
```

Comment:
This was being skipped in RSpec 2, but now is failing the tests in RSpec 3.  To maintain the functionality as it was, just commenting out.